cognite/neat/rules/exporters/_rules2excel.py

```diff
@@ -66,19 +66,30 @@ def export(self, rules: Rules) -> Workbook:
         workbook.remove(workbook["Sheet"])
 
         dumped_rules = rules.model_dump(by_alias=True)
+        if rules.is_reference:
+            # Writes empty reference sheets
+            empty: dict[str, Any] = {field_alias: None for field_alias in dumped_rules["Metadata"].keys()}
+            empty["role"] = (self.output_role and self.output_role.value) or rules.metadata.role.value
+            self._write_metadata_sheet(workbook, empty)
+            self._write_sheets(workbook, {}, rules)
+        else:
+            self._write_metadata_sheet(workbook, dumped_rules["Metadata"])
 
```

Comment:
I am missing `ReferenceMetadata`, can you add it?

Reply:
Yes, I was a bit in doubt about whether we should have it. But can add it np.